corehq/apps/dropbox/views.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from dropbox.client import DropboxOAuth2Flow, DropboxClient
+
+from django.views.generic import View
+from django.http import HttpResponseRedirect, HttpResponse
+
+from .utils import get_dropbox_auth_flow
+
+DROPBOX_ACCESS_TOKEN = 'dropbox_access_token'
+
+
+class DropboxAuthInitiate(View):
+    slug = 'dropbox_auth_initiate'
+
+    def get(self, request, *args, **kwargs):
+        authorize_url = get_dropbox_auth_flow(request.session).start()
+        return HttpResponseRedirect(authorize_url)
+
+
+# TODO AuthCallback
+class DropboxAuthFinalize(View):
+    slug = 'dropbox_auth_finalize'
+
+    def get(self, request, *args, **kwargs):
```

Comment:
this is more or less taken from the dropbox docs. i think it pretty much covers any kind of error we can run into when authenticating